ably/proto/message_test.go

```diff
@@ -172,4 +230,40 @@ var _ = Describe("Message", func() {
 			})
 		})
 	})
+
+	acceptance := func(fixture string, t GinkgoTInterface) {
+		test, cfg := load(fixture, t)
+
+		It("fixture decode", func() {
+			for _, item := range test.Items {
+				err := item.Encoded.DecodeData(nil)
+				Expect(err).NotTo(HaveOccurred())
+				err = item.Encrypted.DecodeData(cfg)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(item.Encrypted.Name).To(Equal(item.Encoded.Name))
+				Expect(item.Encrypted.Data).To(Equal(item.Encoded.Data))
+			}
+		})
+
+		return
+
+		It("fixture encode", func() {
+			for _, item := range test.Items {
+				err := item.Encoded.EncodeData(item.Encrypted.Encoding, cfg)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(item.Encoded.Name).To(Equal(item.Encrypted.Name))
+				Expect(item.Encoded.Data).To(Equal(item.Encrypted.Data))
+			}
+		})
+	}
+
+	Describe("CryptoDataFixtures", func() {
+		Context("with a 128 keylength", func() {
+			acceptance("test-resources/crypto-data-128.json", GinkgoT())
```

Comment:
Hum, it looks like you are trying to use shared examples. I'm not sure I recognise any of the style mentioned in http://onsi.github.io/ginkgo/#shared-example-patterns, especially the usage of `GinkgoT()` which is only mentioned for third party integration.

I suppose you wrote it first with the normal `testing` package and plugged it in this way.